x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/rest/RestExecuteEnrichPolicyAction.java

```diff
@@ -29,6 +29,7 @@ public String getName() {
     @Override
     protected RestChannelConsumer prepareRequest(final RestRequest restRequest, final NodeClient client) throws IOException {
         final ExecuteEnrichPolicyAction.Request request = new ExecuteEnrichPolicyAction.Request(restRequest.param("name"));
+        request.setWaitForCompletion(restRequest.paramAsBoolean("wait_for_completion", true));
```

Comment:
I think we should add `wait_for_completion` parameter to the `enrich.execute_policy.json` rest spec (under `params`).

Reply:
done in 66457ccb9c33d0c7c73af9b4f24c9fc54f9fe835